README.md

```diff
@@ -226,6 +226,24 @@ amioChat.notifications.sendMessagesRead()
 })
 ```
 
+## Postbacks
+
+### postbacks.send(postbackPayload)
+Sends a postback. Postbacks are usually triggered when user presses a button.
```

Comment:
I'd add also something like:
> Buttons are persistent in the chat until the user deletes them. You can expect even a very old postback.

Reply:
That would depend on frontend implementation, not our concern